content/ai/concepts/generative-ai/concepts/images/images.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Images'
+Description: GenAI can generate new images from existing text prompts and images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.
```

Comment:
```suggestion
'Description: GenAI can generate new images from existing text prompts and images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.'
```